test/ros_parameter_handler_test.cpp

```diff
@@ -37,7 +37,6 @@ using namespace psen_scan;
   DELETE_ROS_PARAM("angle_end");                                                                                       \
   DELETE_ROS_PARAM("frame_id");                                                                                        \
   DELETE_ROS_PARAM("skip");                                                                                            \
-  DELETE_ROS_PARAM("publish_topic");                                                                                   \
   DELETE_ROS_PARAM("x_axis_rotation");
```

Comment:
Can we use macros here or is it better to use functions?

Reply:
I think the macros are fine for now.
I moved duplicate code into the constructor. See [Ctor vs Setup](https://github.com/google/googletest/blob/master/googletest/docs/faq.md#should-i-use-the-constructordestructor-of-the-test-fixture-or-setupteardown-ctorvssetup)

Reply:
I think for this purpose in tests, macros are fine.